src/maskAnalysis/utils/getMbrFromPoints.ts

```diff
@@ -91,10 +91,16 @@ export function getMbrFromPoints(points: readonly Point[]): Mbr {
   }
 
   const mbrRotated = rotate(minSurfaceAngle, mbr);
-  const width = mbr[0].column - mbr[2].column;
-  const height = mbr[0].row - mbr[2].row;
+  const width = Math.max(
```

Comment:
Please take care to variable names. This is not the width but the maxSide (or a better name ?).